app/src/main/java/com/tiptopgoodstudio/androidresources/ui/ResourceListActivity.java

```diff
@@ -4,50 +4,43 @@
 import android.content.Intent;
 import android.net.Uri;
 import android.os.Bundle;
-import android.support.v4.app.Fragment;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
-import android.util.Log;
-import android.view.View;
-import android.widget.ProgressBar;
-import android.widget.TextView;
 import android.widget.Toast;
 
 import com.tiptopgoodstudio.androidresources.R;
 import com.tiptopgoodstudio.androidresources.Utilities;
-import com.tiptopgoodstudio.androidresources.db.entity.Resource;
 import com.tiptopgoodstudio.androidresources.ui.adapters.ResourceListAdapter;
 import com.tiptopgoodstudio.androidresources.viewmodel.ResourceViewModel;
 
-import java.util.List;
-
 /**
- * Activity displays Android resource list by user-selected topic
+ * Activity displays Android resource list filtered by user-selected topic
  */
 public class ResourceListActivity extends AppCompatActivity
         implements ResourceListAdapter.ResourceClickListener {
 
-    private static final String TAG = ResourceListActivity.class.getSimpleName();
-
     private RecyclerView mRecyclerView;
     private ResourceListAdapter mResourceListAdapter;
-    private ProgressBar mLoadingIndicator;
-    private TextView mErrorMessageDisplay;
```

Comment:
It's been a while since I made the changes, and maybe I'm not understanding?  There is a Toast that pops up if the URL can't be opened.

Reply:
The TextView mErrorMessageDisplay is not for when URLs cannot be opened. It is to show an error message in case the resources list cannot be retrieved from the DB (instead of showing a blank screen).

Reply:
Is there any reason there would be an error coming from the db?  I'm thinking that if there are no records in the db for the requested topic a LiveData<List<Resource>> object with 0 resources would be returned, yes?  I'm not saying you're wrong, I just can't think of an instance where an error would be returned.

Reply:
I can't think of any specific errors, but thought it would be a good idea to have a generic error message in case of any runtime errors.

Reply:
Totally defer to you.  If you're good with everything else in this PR, why don't I go ahead and merge it in and let you add the error handling.  I'd like to see where you add it.  There are a few places I know absolutely to add it, but always feel like maybe I should be adding more?  I think you have a better instinct for this.